README.md

```diff
@@ -222,6 +223,45 @@ EXAMPLE
 
 Refer to Code: *[src/commands/aem-migration/index-converter.js](./src/commands/aem-migration/index-converter.js)*
 
+## Command: `aio aem-migration:workflow-migrator`
+
+This command migrates asset processing workflows from on-premise or AMS deployments of AEM to processing profiles and OSGi configurations for use in AEM Assets as a Cloud Service..
+
+Please see [here](https://github.com/adobe/aem-cloud-migration/blob/master/README.md)
+ to know more about how the tool works, known limitations and what needs to be handled manually.
+
+### Configurations
+
+Refer to [config](./config/README.md)
+for more info.
+
+```
+USAGE
+  $ aio aem-migration:workflow-migrator
+
+OPTIONS
+  --help                     show help
+
+DESCRIPTION
+  This script will perform an automated migration from custom workflow configurations for asset processing to the corresponding configurations that are required by AEM as a Cloud Service.  After executing the script, the transformed code can be committed to a test branch and deployed to a Cloud Service development environment for testing and validation.
+
+When run, the script will perform the following actions:
+
+#### Create Maven Projects
+Up to Maven projects will be created:
```

Comment:
`The following maven projects will be created:` instead of `Up to Maven projects will be created:`?